src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,44 @@
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  handleTodoReset: (value: Todo | null) => void;
+  selectedUserId: number;
+  selectedTodo: Todo | null;
+};
+
+const defaultUser: User = {
+  id: 0,
+  name: 'Default Username',
+  email: 'Default Email',
+  phone: 'Default Phonenumber',
+};
+
+export const TodoModal: React.FC<Props> = ({
+  handleTodoReset,
```

Comment:
The event handler function `handleTodoReset` can be renamed to adhere to the naming conventions for event handler functions. A recommended name could be `onTodoReset`.

Reply:
....